docs/sudo_cmd_list.txt

```diff
@@ -0,0 +1,80 @@
+FOUND=""; for jar in `find {{search_directories}} -type f -xdev -name \*camel-core\*redhat\*.jar 2>/dev/null | grep fuse | sed -n 's/.*\(redhat-[0-9]\{6\}\).*/\1/p' | sort -u`; do if [ ! -z "${jar}" ]; then if [ ! -z "$FOUND" ]; then FOUND="$FOUND; $jar"; else FOUND=${jar}; fi; fi; done; echo ${FOUND}
+FOUND=""; for jar in `find {{search_directories}} -type f -xdev -name \*cxf-rt\*redhat\*.jar 2>/dev/null | grep fuse | sed -n 's/.*\(redhat-[0-9]\{6\}\).*/\1/p' | sort -u`; do if [ ! -z "${jar}" ]; then if [ ! -z "$FOUND" ]; then FOUND="$FOUND; $jar"; else FOUND=${jar}; fi; fi; done; echo ${FOUND}
+locate karaf.jar | sed -n -e 's/\(.*\)lib\/karaf\.jar$/\1/p' | xargs -n 1 --no-run-if-empty readlink --canonicalize
+find {{search_directories}} -xdev -type f -name karaf.jar 2> /dev/null | sed -n -e 's/\(.*\)lib\/karaf\.jar$/\1/p' | xargs -n 1 --no-run-if-empty readlink --canonicalize | sort -u
+ls -1 "{{ item }}"/bin/fuse 2>/dev/null
+ls -1 "{{ item }}"/system/org/jboss 2>/dev/null
+/usr/bin/systemctl list-unit-files --no-pager
+/sbin/chkconfig --list
+locate camel-core | grep fuse | sed -n 's/.*\(redhat-[0-9]\{6\}\).*/\1/p'
+ls -1 '{{ item }}/system/org/apache/camel/camel-core' 2>/dev/null | grep fuse | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
+locate activemq | grep fuse | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
+ls -1 '{{ item }}/system/org/apache/activemq/activemq-camel' 2>/dev/null | grep fuse | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
+locate cxf-rt | grep fuse | sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
+ls -1 '{{ item }}/system/org/apache/cxf/cxf-rt-bindings-coloc' 2>/dev/null | grep fuse |sed -n 's/^.*\(redhat-[0-9]\{6\}\).*/\1/p'
+yum repolist enabled 2> /dev/null
+subscription-manager facts --list | grep '^cpu.cpu(s).' | sed -n -e 's/^.*cpu.cpu(s).\s*//p'
+subscription-manager facts --list | grep '^cpu.core(s)_per_socket.' | sed -n -e 's/^.*cpu.core(s)_per_socket.\s*//p'
+subscription-manager facts --list | grep '^cpu.cpu_socket(s).' | sed -n -e 's/^.*cpu.cpu_socket(s).\s*//p'
+subscription-manager facts --list | grep '^virt.host_type.' | sed -n -e 's/^.*virt.host_type.\s*//p'
+subscription-manager facts --list | grep '^virt.is_guest.' | sed -n -e 's/^.*virt.is_guest.\s*//p'
+subscription-manager facts --list 2>/dev/null | grep '^virt.uuid.' | sed -n -e 's/^.*virt.uuid.\s*//p'
+LANG=C subscription-manager status | grep -e 'Overall Status:'| cut -d ":" -f2
+subscription-manager list --consumed 2>/dev/null | grep -e '^SKU' -e '^Subscription Name' | sed -n -e 's/SKU\s*.\s*//p' -e 's/Subscription Name\s*.\s*//p' | awk '{ ORS = (NR%2 ? " - " {{":"}} RS) } 1'
+subscription-manager identity 2>/dev/null | 
+grep 'system identity:' |
+sed 's/system identity:\s*//'
+
```

Comment:
I just noticed this blank line. I wonder if there's a bug in `collect_sudo_commands` that results in something adding an empty string to `list_cmds`. I have an idea that I'll comment in that function…